### DISCUSSION TITLE
What is WIREGUARD_ADDRESSES for Mullvad wireguard configuration?

### DISCUSSION THREAD

**Discussion by karipov** [qdm12/gluetun#988] [Q&A] [answered]
Hi! I'm reading the wiki and I'm not exactly sure what `WIREGUARD_ADDRESSES` stands for.

- How do I obtain this address(es)?

From the [wiki page](https://github.com/qdm12/gluetun/wiki/Mullvad) I don't understand the wording of:

> "WIREGUARD_ADDRESSES is your IP network interface address in the format xx.xx.xx.xx/xx. Note this is specific by Wireguard keypair and the same for all servers."

I'd really appreciate the help! I'm running gluetun through portainer on an ubuntu server.

**Comment by qdm12** [comment, marked as answer]
Download a Wireguard configuration from the Mullvad website, and you'll find an `Address` field in the `[Interface]` section in the file; that's the one you need to use. It's the private IP address used in the virtual private network on the Mullvad server.

**Comment by blade10101** [comment]
there is no config to download, only a key you create
